[docs] link fix and minor edits to the global apps sections (#18161)
yugabyte/yugabyte-db: docs/content/preview/develop/build-global-apps/_index.md

> ### Data Residency for Compliance

Reviewer: ```suggestion
### Data residency for compliance
```

Applications read from source of truth, possibly with higher latencies.|

|[Duplicate indexes](./duplicate-indexes)|
{{<header Level="6">}} Consistent data everywhere {{</header>}}
Set up covering indexes with schema the same as the table in multiple regions to read immediately consistent data locally.|

|[Active&#8209;active single&#8209;master](./active-active-single-master)|
{{<header Level="6">}} Secondary database that can serve reads {{</header>}}
Set up a second cluster that gets populated asynchronously and can start serving data in case the primary fails. Can also be used for [blue/green](https://en.wikipedia.org/wiki/Blue-green_deployment) deployment testing.|

|[Active&#8209;active multi&#8209;master](./active-active-multi-master)|
{{<header Level="6">}} Two clusters serving data together {{</header>}}
Two regions or more, manual failover, a few seconds of data loss (non-zero RPO), low read/write latencies, some caveats on transactional guarantees.|

|[Latency&#8209;optimized geo&#8209;partitioning](./latency-optimized-geo-partition)|
{{<header Level="6">}} Fast local access {{</header>}}

> Partition your data and place them in a manner that the data belonging to nearby users can be accessed faster.|

Reviewer: ```suggestion
Partition your data and place it such that the data belonging to nearby users can be accessed faster.|
```

|[Locality&#8209;optimized geo&#8209;partitioning](./locality-optimized-geo-partition)|
{{<header Level="6">}} Local law compliance {{</header>}}

> Partition your data and place them in a manner that the rows belonging to different users will be located in their respective countries.|

Reviewer: ```suggestion
Partition your data and place it such that the rows belonging to different users are located in their respective countries.|
```
